docs/core/tools/global-tools.md

```diff
@@ -126,33 +126,47 @@ The .NET Core CLI tries to add the default locations to the PATH environment var
 * On macOS, if you've installed the .NET Core SDK using *.tar.gz* files and not *.pkg*.
 * On Linux, you need to edit the shell environment file to configure the PATH.
```

Comment:
> If you've set the DOTNET_SKIP_FIRST_TIME_EXPERIENCE environment variable.

Note, the big flag that controls many things is removed and a replacement for this specific functionality is DOTNET_ADD_GLOBAL_TOOLS_TO_PATH

> On Linux, you need to edit the shell environment file to configure the PATH.

If it is installed via deb and rpm and on bash you don't need this